features/step_definitions/borrower_views_deed.rb

```diff
@@ -107,3 +107,57 @@
   page.should have_content("[Awaiting confirmation from #{f_name}"\
                              "#{m_name} #{s_name}]")
 end
+
+Given(/^I create a deed where the Lender is Coventry Building Society$/) do
+  step %(I have valid deed data with <#{borrower}> borrowers)
+  step %(I create the deed via the Deed API)
+  step %(the deed id is returned by the Deed API)
+end
```

Comment:
This step should probably be removed as it is no longer being called from the scenario.